src/utils/uintarray-encoding.js

```diff
@@ -162,16 +300,24 @@ function encodingCharToNumber(x: string): number {
   }
 }
 
-export function stringToUintArray(s: string): number[] {
-  const array = [];
-  let val = 0;
-  for (let i = 0; i < s.length; i++) {
-    const x = encodingCharToNumber(s[i]);
-    val = (val << 5) + (x & 0b11111);
-    if ((x & 0b100000) === 0) {
-      array.push(val);
-      val = 0;
-    }
+function decodeUint(
```

Comment:
Can you please add a comment explaining the possible return values?

Reply:
done